requirements.txt

```diff
@@ -1,49 +1,48 @@
 # LITELLM PROXY DEPENDENCIES #
 anyio==4.4.0 # openai + http req.
-openai==1.40.0 # openai req. 
```

Comment:
this is a very large amount of changes to our requirements.txt. Can this be stacked into smaller changes. I'd like each change to run through our load testing 

Could you stack 2-3 prs for this @Jacobh2 ?

Reply:
For sure! Any particular grouping you have in mind?